### ISSUE TITLE
Jupyter Notebook Kernel keeps dead

### ISSUE THREAD

**Issue by sasi143** [rapidfuzz/RapidFuzz#26]
```python
from rapidfuzz import fuzz
from rapidfuzz import process

for i in cat_no:
    final = process.extractBests(i,sent,score_cutoff=80)
    if len(final) != 0:
        print(i)
        print(final)
```
I am running above code in my Jupiter notebook, when I am looping **"process.extractBests"** more than 1000 times then my Jupiter kernal keep saying it is dead and some random popup coming and saying **python program stop working**.

can you please help me out

**Comment by maxbachmann** [comment]
Are you working with the latest version of rapidfuzz?
If your working with 0.7.5 then there was a known issue: https://github.com/maxbachmann/rapidfuzz/issues/25 that is fixed in v0.7.6

**Comment by sasi143** [comment]
@maxbachmann  Yes, After upgrading to the new version it got sorted. Thanks a lot

Closing the issue
